backend/core/interfaces/docker_repository.py

```
@@ -0,0 +1,295 @@
+"""Docker repository interfaces"""
+
+from abc import ABC, abstractmethod
+from typing import List, Optional, Dict, Any
+from ..entities.docker_entity import DockerContainer, DockerImage, DockerNetwork, DockerComposeStack
+from ..value_objects.host_connection import HostConnection
```

Ruff (0.11.9)

`..value_objects.host_connection.HostConnection` imported but unused

Remove unused import: `..value_objects.host_connection.HostConnection`

(F401)